drivers/flash/Kconfig.at45

```diff
@@ -29,6 +29,12 @@ config SPI_FLASH_AT45_USE_READ_MODIFY_WRITE
 	  Main Memory Program without Built-In Erase (opcode 0x02). This allows
 	  writing of data without prior erasing of corresponding pages.
 
+config SPI_FLASH_AT45_QUIRK
+	bool "Use quirk mode"
+	help
+	  For some AT45 devices that do not work with certain commands that combined
+	  a few steps together, enable this quirk mode to use workaround.
```

Comment:
Should this be in devicetree?

Reply:
oh yes! that thought slipped right past my mind!

Reply:
thanks!